src/components/PassphraseDialog/PassphraseModal/PassphraseModal.scss

```diff
@@ -0,0 +1,54 @@
+@import "constants/style";
+$color-form-background--dark: #292f3b;
+
+.password-modal {
+  position: absolute;
+  backdrop-filter: blur(4px);
+  width: 100%;
+  height: 100%;
+}
+
+.password-modal__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.password-modal__container {
+  padding: $spacing--lg;
+  background: #ffffff;
+  border-radius: 0.5em;
+  box-shadow: 0 10px 20px rgba(black, 0.2);
```

Comment:
you can use colors from `colors.scss` instead of hardcoded hex values or named standard colors (white being `$gray--000`, for example).